cloud-controller-manager/do/loadbalancers.go

```diff
@@ -82,7 +82,7 @@ const (
 
 	// defaultActiveTimeout is the number of seconds to wait for a load balancer to
 	// reach the active state.
-	defaultActiveTimeout = 90
+	defaultActiveTimeout = 120
```

Comment:
To add a bit of a buffer between when we'll choose to delete an LB.

Reply:
It might make sense to include the buffer rationale into the GoDoc so that it's visible beyond this PR.